modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -22,22 +22,21 @@ endif::[]
 
 :_mod-docs-content-type: PROCEDURE
 [id="installation-adding-registry-pull-secret_{context}"]
-= Configuring credentials that allow images to be mirrored
+= Configuring credentials to enable mirroring images
 
-Create a container image registry credentials file that allows mirroring
-images from Red Hat to your mirror.
+Follow the procedure to create a container image registry credentials file for mirroring images from Red Hat to your mirror registry.
 
 ifdef::restricted[]
 [WARNING]
 ====
-Do not use this image registry credentials file as the pull secret when you install a cluster. If you provide this file when you install cluster, all of the machines in the cluster will have write access to your mirror registry.
+Do not use this image registry credentials file as the pull secret when installing a cluster. Using this file will grant write access to your mirror registry for all machines in the cluster.
 ====
 endif::restricted[]
 
 ifdef::restricted[]
 [WARNING]
 ====
-This process requires that you have write access to a container image registry on the mirror registry and adds the credentials to a registry pull secret.
+For this process, you need write access to the mirror registry. You also need to include the credentials in a registry pull secret.
```

Comment:
SSG says to avoid stacking admonitions. Could this 2nd one just become a Prerequisites bullet?

Reply:
Added the warning in prereq:
* You have write access to the mirror registry. 
* You included the credentials in the registry pull secret.